packages/vector/src/services/pgvector-provider.ts

```
@@ -44,137 +49,151 @@ export class PgVectorProvider implements VectorProvider {
+		try {
+			// Get or create collection
+			const collectionId = await this.getOrCreateCollectionId(
+				client,
+				options.collection,
+			);
+
+			// Generate embeddings
+			const embeddings = await this.generateEmbeddings(options.documents);
+
+			// Prepare values for bulk insert
+			const values: any[] = [];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)